index.html

```diff
@@ -240,13 +240,86 @@ <h3 class="language-container-text">Skills</h3>
       </div>
     </section>
     <!-- Footer section -->
-    <section class="footer">
-      <div class="last-msg">
-        <p class="last-msg-text">
-          I'm always interested in hearing about new projects, so if you'd like
-          to chat please get in touch.
-        </p>
+    <footer>
+      <div class="footer-desktop-container">
+        <div class="last-msg">
+          <p class="last-msg-text">
+            I'm always interested in hearing about new projects, so if you'd like
+            to chat please get in touch.
+          </p>
+        </div>
+        <form
+          action="https://formspree.io/f/xaykrgow"
+          method="post"
+          id="form"
+          class="contact-form"
+        >
+          <input
+            type="text"
+            class="ftext"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="email"
+            class="femail"
+            placeholder="Email address"
+            required
+          />
+          <textarea
+            name="message"
+            id="message"
+            class="ftextarea"
+            maxlength="500"
+            placeholder="Enter text here"
+            required
+          ></textarea>
+          <div class="footer-btn">
+            <button class="get-in-touch click" type="submit">Get in touch</button>
+          </div>
+        </form>
```

Comment:
Kindly check and ensure that the image close to the right side of the form is not too close to the textarea field in order to match the actual design.